pkg/cue/model/instance.go

```diff
@@ -99,49 +95,19 @@ func (inst *instance) Unify(other Instance, options ...sets.UnifyOption) error {
 
 // NewBase create a base instance
 func NewBase(v cue.Value) (Instance, error) {
-	vs, err := openPrint(v)
-	if err != nil {
-		return nil, err
-	}
 	return &instance{
-		v:    vs,
+		v:    v,
 		base: true,
 	}, nil
 }
 
 // NewOther create a non-base instance
 func NewOther(v cue.Value) (Instance, error) {
-	vs, err := openPrint(v)
-	if err != nil {
-		return nil, err
-	}
 	return &instance{
-		v: vs,
+		v: v,
 	}, nil
 }
 
-func openPrint(v cue.Value) (string, error) {
```

Comment:
if we don't use this function, how do we detect the error now? can we still aware of the error detected like blew 

```
errInfo, contain := IndexMatchLine(string(ret), "_|_")
	if contain {
		return "", errors.New(errInfo)
	}
```

Reply:
The error will be reported in `ret.Err`, we do not need `IndexMatchLine` anymore.

Reply:
The error will be reported in `ret.Err`, we do not need `IndexMatchLine` anymore.

Reply:
please remove that dead code.